### DISCUSSION TITLE
How to convert t5xxl_fp8_e4m3fn.safetensors into diffusers format?

### DISCUSSION THREAD

**Discussion by 8-unlimit** [huggingface/diffusers#9657] [Q&A] [answered]
How to convert` t5xxl_fp8_e4m3fn.safetensors` into diffusers format?
I didn't find the relevant code in the diffusers script, so I'm very curious about this. Can anyone give me an answer?🙇‍
By the way, if flux1-dev-fp8.safetensors is converted into diffusers format, can I directly modify the following code to look like this?
`dtype = torch.float8_e4m3fn`
https://github.com/huggingface/diffusers/blob/6a5f06488c0d88c1827c016835cd5f64abe4b52c/scripts/convert_flux_to_diffusers.py#L46

**Comment by a-r-r-o-w** [comment, marked as answer]
I think you might find the comments in this issue insightful, as well as a solution on how to load it: https://github.com/huggingface/diffusers/issues/9165

cc @sayakpaul for Flux
